contracts/TokenVesting.sol

```diff
@@ -51,7 +51,13 @@ contract TokenVesting is Ownable {
         return vestings[_vestingId].amount;
     }
 
+    function removeVesting(uint256 _vestingId) public onlyOwner {
+        require(vestings[_vestingId].beneficiary != address(0x0) && !vestings[_vestingId].released);
+        vestings[_vestingId].released = true;
+    }
+
     function addVesting(address _beneficiary, uint256 _releaseTime, uint256 _amount) public onlyOwner {
+        require(_beneficiary != address(0x0), "ADD_BENEFICIARY");
```

Comment:
Can we use `ERC-1066` instead of hardcoded strings for error messages?

Reply:
yeah, good stuff will explore and migrate if applicable to our needs.